platform-sdk/swirlds-logging/src/main/java/com/swirlds/logging/api/internal/LoggingSystem.java

```diff
@@ -217,7 +217,7 @@ public void installProviders() {
                 .filter(LogProvider::isActive)
                 .toList();
         providers.forEach(p -> p.install(getLogEventFactory(), this));
-        EMERGENCY_LOGGER.log(Level.DEBUG, providers.size() + " logging providers installed: " + providers);
+        EMERGENCY_LOGGER.log(Level.TRACE, providers.size() + " logging providers installed: " + providers);
```

Comment:
Won't this increase the volume of log data (once tests are re-enabled). 
Might there also be any performance impact ?

Reply:
TRACE is a lower level than DEBUG and therefore the log won't be shown by default.
That is called only at instantiation so there is no performance impact.
@mxtartaglia-sl next to that I do not get why you reduced the level. That line will maybe called 1 time per test and based on that it should not be a problem.

Reply:
Yes, in a second thought this change is not needed. The only problem is that many tests have the debug level enabled by default, making this line repeat often enough. Maybe we should address the test later and leave  the level as it is.

Reply:
Rollbacked